scripts/diag/tmux-vs-rawpty-idle.py

```
@@ -0,0 +1,209 @@
+    ap.add_argument("--reps", type=int, default=4)
+    ap.add_argument("--gaps", default="0.1,1,5,20")
+    a = ap.parse_args()
+    gaps = [float(g) for g in a.gaps.split(",")]
+
+    if not os.path.exists(a.socket):
+        print(f"no such tmux socket: {a.socket}")
+        return 1
+
+    subprocess.run([TMUX, "-S", a.socket, "kill-session", "-t", f"={SESSION}"],
+                   capture_output=True)
+    subprocess.run([TMUX, "-S", a.socket, "new-session", "-d", "-s", SESSION,
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)